scripts/player_movement.gd

```diff
@@ -0,0 +1,31 @@
+class_name Player extends CharacterBody2D
+
+
+@export var MAX_SPEED: int = 200
+@export var JUMP_VELOCITY: int = -300.0
+@export var ACCELERATION: int = 1500
+@export var DECELERATION_TIME: float = 0.2
```

Comment:
This should be limited to >0 diving by zero is usually frowned upon though it will cause deceleration to be ♾️

Reply:
@aaronspringer I'd look at this resource https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_exports.html

Specifically something close to: 

@export_range(0, 100, 1, "or_greater") var DECELERATION_TIME: float = 0.2